sql/sql.py

```diff
@@ -0,0 +1,99 @@
+from typing import Literal
+
+import discord
+from redbot.core import commands
+from redbot.core.bot import Red
+from redbot.core.config import Config
+
+# MySQL
+import mysql.connector
+
+RequestType = Literal["discord_deleted_user", "owner", "user", "user_strict"]
+
+
+class SQL(commands.Cog):
+    """
+    SQL cog that can be used for interacting with multiple databases.
+    """
+
+    __version__ = "1.0.1"
+
+    def __init__(self, bot: Red) -> None:
+        self.bot = bot
+        self.config = Config.get_conf(
+            self,
+            identifier=572944636209922059,
+            force_registration=True,
+        )
+        self.startup_task = self.bot.loop.create_task(self.startup())
+
+    async def startup(self):
+        mysql_cred = await self.bot.get_shared_api_tokens("mysql")
+        self.sqql = mysql.connector.connect(user=mysql_cred.get("username"), password=mysql_cred.get("password"), database="")
+        self.cursorr = self.sqql.cursor()
+        self.cursorr.execute("SET @@wait_timeout = 31536000")
+
+    def cog_unload(self):
+        self.startup_task.cancel()
+    
+    def format_help_for_context(self, ctx):
+        """Thanks Sinbad!"""
+        pre_processed = super().format_help_for_context(ctx)
+        n = "\n" if "\n\n" not in pre_processed else ""
+        return f"{pre_processed}{n}\nCog Version: {self.__version__}"
+
+
+    async def red_delete_data_for_user(self, *, requester: RequestType, user_id: int) -> None:
+        # TODO: Replace this with the proper end user data removal handling.
+        super().red_delete_data_for_user(requester=requester, user_id=user_id)
+    
+    
+    @commands.is_owner()
+    @commands.group()
+    async def sql(self, ctx):
+        """SQL commands"""
+        pass
+
+    @sql.command()
+    async def version(self, ctx):
+        """Check what version of the SQL cog you have."""
+        await ctx.reply(f"This cog is on version {self.__version__}.", mention_author=False)
+
+    @sql.group()
+    async def mysql(self, ctx):
+        """MySQL commands"""
+        pass
+
+    @mysql.group()
+    async def delete(self, ctx):
+        """Delete database/table"""
```

Comment:
At the moment this only has one subcommand. If you don't plan on adding any more subcommands then I would suggest just making this `[p]sql mysql delete`